optuna/study.py

```diff
@@ -35,12 +33,17 @@
     ObjectiveFuncType = Callable[[trial_module.Trial], float]
```

Comment:
Could you remove the 31st line (`from optuna.distributions import BaseDistribution  # NOQA`), please?
I found that `BaseDistribution` is not used in `study.py` during the review although it seems to be unrelated to this PR directly.

Reply:
Thank you for your careful reviewing.
Removed: https://github.com/pfnet/optuna/pull/470/commits/16cc5fe3d90bce13544d2a6b156e30b2ad838ef5